FreeRTOS/FreeRTOS-Plus-TCP
# Uncrustify PR Check Update

Uncrustify PR Check Update

Description
-----------
Change to uncrustify to show files that need to be updated and create a cleaner log on runs

Test Steps
-----------
Ran the workflow after the changes to ensure it worked, [failed run one](https://github.com/Skptak/FreeRTOS-Plus-TCP/actions/runs/5549996475/jobs/10134669506#step:4:577), [failed run two](https://github.com/Skptak/FreeRTOS-Plus-TCP/actions/runs/5549885222/jobs/10134447280#step:4:577), and a [passing run](https://github.com/Skptak/FreeRTOS-Plus-TCP/actions/runs/5550053863/jobs/10134787257#step:4:577)

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
